dataframe-excel/src/main/kotlin/org/jetbrains/kotlinx/dataframe/io/xlsx.kt

```diff
@@ -255,6 +271,8 @@ public fun DataFrame.Companion.readExcel(
  * when set to false, it operates as [NameRepairStrategy.MAKE_UNIQUE],
  * ensuring unique column names will make the columns be named according to excel columns, like "A", "B", "C" etc.
  * for unstructured data.
+ * @param parseEmptyAsNull when set to true, empty strings in cells are parsed as null (default true).
```

Comment:
There may be more representations in strings that can refer to `null`. maybe it might be better to follow readCsv and other parsing options for this. It will be overkill to add the full ParserOptions as argument (since datetime and locales are already handled by excel), but csv, for instance uses `nullStrings: Set<String>= setOf("", "NA", "N/A", "null", "NULL", "None", "none", "NIL", "nil")`, so it might make sense to add a `nullStrings` argument here as well

Reply:
Created an issue:
https://github.com/Kotlin/dataframe/issues/1248

Reply:
I don't think is a good idea to introduce a new non-trivial logic just before release, need more time for implementation and testing.

Reply:
okay sure, makes sense